vxtelegram/telegram.py

```diff
@@ -187,12 +177,19 @@ def handle_raw_inbound_message(self, message_id, request):
             message_id=message_id,
             content=message['content'],
             to_addr=message['to_addr'],
+            to_addr_type='telegram_username',
             from_addr=message['from_addr'],
+            from_addr_type='telegram_id',
```

Comment:
@hodgestar that's definitely possible. Telegram's updates contain a `from` field which has an `id` (the `telegram_id`), `username`, `first_name` and `last_name`. I'll create an issue and make a PR for it :)